test/cloudtest/pkg/commands/main.go

```diff
@@ -281,23 +280,21 @@ func (ctx *executionContext) performExecution() error {
 			switch event.kind {
 			case eventClusterUpdate:
 				ctx.performClusterUpdate(event)
-				ctx.printStatistics()
 			case eventTaskUpdate:
 				// Remove from running onces.
 				ctx.processTaskUpdate(event)
 			}
-		case <-timectx.Done():
-			ctx.printStatistics()
-			timectx, cancelfunc = context.WithTimeout(context.Background(), 1*time.Minute)
-			defer cancelfunc()
 		case <-termChannel:
 			return errors.New("termination request is received")
 		case <-timeoutCtx.Done():
 			ctx.printStatistics()
 			return errors.Errorf("global timeout elapsed: %v seconds", ctx.cloudTestConfig.Timeout)
+		case <-time.After(60 * time.Second):
```

Comment:
This is not right, @alex-yust you do same mistake as I, since updates happen more often it will lead to update very-very often, and will no print messages, since will exit with event := <-ctx.operationChannel and not with print timeout.

Reply:
Yeah, I understood that a bit later too ;)

Reply:
- [ ] create dedicated timer for calling printStatistics